src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/IssuedTokenParametersElement.cs

```diff
@@ -0,0 +1,328 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Configuration;
+using System.IO;
+using System.Text;
+using System.Xml;
+using CoreWCF.IdentityModel.Tokens;
+using CoreWCF.Runtime;
+using CoreWCF.Security.Tokens;
+
+namespace CoreWCF.Configuration
+{
+    internal sealed class IssuedTokenParametersElement : ServiceModelConfigurationElement
+    {
```

Comment:
This was public on .NET Framework, was there a reason to make this internal?

Reply:
I made this Internal because it was not fully implemented.  Nor have tests been written for it.

Reply:
Set it to public